Solutions/Vectra AI Detect/Analytic Rules/VectraDetect-Suspected-Behavior-by-Tactics.yaml

```diff
@@ -39,6 +39,7 @@ query: |
   | project-rename threat_score = FlexNumber1
   | project-rename certainty_score = FlexNumber2
   | project-rename triaged = DeviceCustomString5
+  | where triaged != "True" and AdditionalExtensions !contains "account"
```

Comment:
Can this "contains" be replaced with "has"?

Reply:
done in all templates